ResizeImage.py

```diff
@@ -47,7 +47,10 @@ def resize_image(src):
     _, outfile = mkstemp()
     try:
         image = Image.open(src)
-        log.debug("Resizing: %s, original size: %s", src, image.size())
+        log.debug("Resizing: %s, original size: %dx%d",
```

Comment:
File size is logged on line 24, which has the size as uploaded to the S3 bucket.

`Got new file: calligre-images-pending-resize:facebook-10153352120094958-Q751W, size 112975`